app/frontend/src/AppRoute.tsx

```diff
@@ -53,12 +58,20 @@ export default function AppRoute({
       render={({ location }) => (
         <>
           {isAuthenticated ? (
-            <Container className={classes.standardContainer}>
+            <Container
+              className={classNames({
+                [classes.fullWidthContainer]: variant === "full-width",
+                [classes.fullscreenContainer]: variant === "full-screen",
```

Comment:
We haven't got a use case for a full screen container yet for authenticated routes, but we may as well fully support it since this doesn't break existing behaviour? Also it'd be more accurate to the behaviour described by the prop, which wasn't the case before as is only full screen when used on unauthenticated routes...

Reply:
Never mind, looks like we do a have full screen use case since the latest merge conflict so this is a valid case to support :D